src/playback/playback.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import DashboardPage from '../utils/DashboardPage';
+import PlaybackSummary from './summary';
+import Section from '../utils/Section';
+
+export default class Playback extends React.Component {
+  render() {
+    return (
+      <DashboardPage
+        title="Playback"
+        subtitle="Release criteria"
+      >
+        <Section title="Firefox (Desktop)">
+          <Grid container spacing={24}>
+            <Grid item xs={6} key="1">
+              <PlaybackSummary
+                key="VP9_firefox"
+                bits={64}
+                encoding="VP9"
+                browserId="firefox"
+              />
+            </Grid>
+            <Grid item xs={6} key="2">
+              <PlaybackSummary
+                key="H264_firefox"
+                bits={64}
+                encoding="H264"
+                browserId="firefox"
+              />
+            </Grid>
+          </Grid>
+        </Section>
+        <Section title="Firefox (Android)">
+          <Grid container spacing={24}>
+            <Grid item xs={6} key="3">
+              <PlaybackSummary
+                key="VP9_geckoview"
+                bits={64}
+                encoding="VP9"
+                browserId="geckoview"
+              />
+            </Grid>
+            <Grid item xs={6} key="4">
+              <PlaybackSummary
+                key="H264_geckoview"
+                bits={64}
+                encoding="H264"
+                browserId="geckoview"
+              />
+            </Grid>
+          </Grid>
+        </Section>
+        <Section title="Firefox Preview (Android)">
+          <Grid container spacing={24}>
+            <Grid item xs={6} key="5">
+              <PlaybackSummary
+                key="VP9_fenix"
+                bits={64}
+                encoding="VP9"
+                browserId="fenix"
+              />
+            </Grid>
+            <Grid item xs={6} key="6">
+              <PlaybackSummary
+                key="H264_fenix"
+                bits={64}
+                encoding="H264"
+                browserId="fenix"
+              />
+            </Grid>
+          </Grid>
+        </Section>
+      </DashboardPage>
+    );
+  }
+}
```

Comment:
This is hard-coded, but that can be fixed later.

Reply:
This is what I wanted to fix. To not have them hard-coded and to avoid duplication of the keys for some elements. It can be merged right now if you are ok with fixing it later

Reply:
@airimovici yes, if you intended to make this more dynamic, then please attempt to do so.

Reply:
... the dynamic nature will catch the missing win32

Reply:
I partially did it, I'll push the working changes on Monday.
@davehunt @klahnakoski Sorry for not being able to join yesterday meeting

Reply:
@airimovici I hope you are able to push what you have.  If it is incomplete (`git push --no-verify`), and I can give you suggestions.

Reply:
@klahnakoski I pushed the changes, but there are some problems I'm investigating cases where it either shows no data when it should show data(H264 encoded videos >1080p: there are no video tests with 60 frames ex: ) or where we get extra data that we should not get(like >1080p results for g5 tests). I think this can be merged now and I'm going to add issues for the problems I discovered.

Reply:
![playback-summary-page](https://user-images.githubusercontent.com/51707553/65523095-6cc8db80-def4-11e9-9575-b123bf220528.png)